test/integration/cli-nics.test.js

```diff
@@ -27,7 +28,6 @@ var NIC2;
 
 var testOpts = {
     skip: (
-        (!process.env.TEST_KNOWN_FAIL && 'known failure, see TRITON-1398') ||
         (!h.CONFIG.allowWriteActions && 'requires config.allowVolumesTests')
```

Comment:
Should that string be `'requires config.allowWriteActions'` instead of `'requires config.allowVolumesTests'`?

Reply:
Actually yes, you're right. It went in that way in commit 18b8fae3. I guess it must have been copy/pasted from elsewhere and they forgot to change it.

Reply:
Nice catch. Looks like that was done incorrectly originally. Fixed this.